spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -74,6 +74,10 @@
   describe "#destroy_collection" do
     let(:user) { create(:user) }
 
+    let!(:empty_adminset) { create(:admin_set, title: ['Empty Admin Set'], edit_users: [user], creator: [user.user_key], with_permission_template: true) }
+    let!(:adminset) { create(:admin_set, title: ['Admin Set with Work'], edit_users: [user], creator: [user.user_key], with_permission_template: true) }
```

Comment:
The `let!` isn't required here `let` would be fine because `adminset` is referenced on line 79